src/north/influxdb/InfluxDB.schema.jsx

```diff
@@ -1,4 +1,5 @@
 import React from 'react'
+import influxdb from './influxDB.png'
```

Comment:
For consistency with other connectors, can you rename the image file InfluxDB.png and call this variable InfluxDB ?